### ISSUE TITLE
Comments to "Hosting Azure Functions in Google Cloud Run"

### ISSUE THREAD

**Issue by mikhailshilkov** [mikhailshilkov/mikhailio-hugo#33]
Add your comment to [Hosting Azure Functions in Google Cloud Run](https://mikhail.io/2020/02/azure-functions-in-google-cloud-run/).
The comments will be displayed directly on the page.
I may edit or merge your comments in a non-significant way to make sure they are displayed properly on the website.

**Comment by aminelaadhari** [comment]
@mikhailshilkov thanks for the article, do you think it works as well with durable functions?

**Comment by mikhailshilkov** [comment]
No. Durable Functions rely on Azure Storage queues and table which aren't available in Google Cloud.

**Comment by aminelaadhari** [comment]
@mikhailshilkov thanks for your response. I thought we can switch durable function storage to something else like redis: https://github.com/Azure/azure-functions-durable-extension/blob/f55c239793a1778ff5863df002de22b356d149be/test/FunctionsV2/RedisDurabilityProviderFactory.cs#L9
https://github.com/Azure/azure-functions-durable-extension/blob/f9a43f80f5bdc50c10c06eaa5de7ac9c241132fd/test/FunctionsV2/PlatformSpecificHelpers.FunctionsV2.cs#L68
https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-bindings#hostjson-settings

**Comment by mikhailshilkov** [comment]
Redis backend is not officially supported. I think they missed some features in Redis compared to Azure Storage. Maybe you could if you are willing to take the risks of running in an unsupported mode. I wouldn't use it for production though.
